sphinx/src/orga/00_intro.rst

```diff
@@ -1,12 +1,7 @@
-Organisation Documentation
-##########################
+# Organization
 
 .. toctree::
+
   guidelines
-  governance
-  subteams
-  joining-the-team
   minutes/00_intro
-  cfep-index
-  getting-in-touch
   funding/00_intro
```

Comment:
We should move this section too if possible. It can be done in a subsequent PR if you prefer, though. 🙏

Reply:
Ok